.github/workflows/pr.yml

```diff
@@ -17,6 +17,7 @@ jobs:
             **Note for the maintainer:**  Commands available:
 
             * `/run-acceptance-tests` - used to test run the acceptance tests for the project
+            * `/run-integration-tests` - used to run the full integration tests for the project
```

Comment:
What'd be the difference between acceptance and integration tests for us? Do we document this?